packages/playwright-core/src/server/registry/dependencies.ts

```diff
@@ -290,7 +290,9 @@ async function executablesOrSharedLibraries(directoryPath: string): Promise<stri
     const basename = path.basename(filePath).toLowerCase();
     if (isSharedLib(basename))
       return filePath;
-    if (await checkExecutable(filePath))
+    if (process.platform !== 'win32' && await checkExecutable(filePath))
```

Comment:
Would it make sense to change checkExecutable to do the right thing on windows instead?